src/main/java/com/cloudbees/hudson/plugins/folder/relocate/StandardHandler.java

```diff
@@ -51,7 +52,10 @@
 @Extension(ordinal=-1000) public final class StandardHandler extends RelocationHandler {
 
     public HandlingMode applicability(Item item) {
-        if (item instanceof TopLevelItem && item instanceof AbstractItem && item.getParent() instanceof DirectlyModifiableTopLevelItemGroup && !validDestinations(item).isEmpty()) {
+        if (item instanceof TopLevelItem
+                && item instanceof AbstractItem
+                && item.getParent() instanceof DirectlyModifiableTopLevelItemGroup
```

Comment:
If you are a valid target for moving into then you must implement `Does BCT support package annotations BTW?`